src/scripts/ic8/shared/common_checks.py

```diff
@@ -19,9 +19,6 @@ def __init__(self, *args, **kwargs):
         params = self.parameters
 
         # Gather the expectations for this class:
-        self.EXPECTED_CF_HISTORICSCENARIOS = params.get_historiccounterfactuals().index.to_list()
-        self.EXPECTED_CF_SCENARIOS = params.get_counterfactuals().index.to_list()
-        self.EXPECTED_FUNDING_SCENARIOS = params.get_scenarios().index.to_list()
         self.EXPECTED_INDICATORS = params.get_indicators_for(self.disease_name)
         self.EXPECTED_EPI_INDICATORS = self.EXPECTED_INDICATORS.loc[self.EXPECTED_INDICATORS.use_scaling].index.to_list()
```

Comment:
an equation is being formed here between `use_scaling` and `epi-indicator` which i can understand, but just flagging it.

Reply:
Thank you @tbhallett I actually noted that as an issue in our old repo. I will change that today or when I am back so that there is no confusion later on!